### ISSUE TITLE
Improve performance of overpass.osmcha.org and / or enable switching overpass providers

### ISSUE THREAD

**Issue by tordans** [OSMCha/osmcha-frontend#725]
I see in my network tab that it takes 11 seconds for the overpass request to finish before I can see the changes.
It looks like the overpass server at `overpass.osmcha.org` is overloaded.

A kind of workaround would be to allow to change the servers in the frontend.

But of course a better way would be to have a server that can handle the load.
I read in https://www.openstreetmap.org/user/wille/diary/403465#comment56731 that this is a know problem. 
> We need to do more work on it, as the performance of the API is not good yet. We had to reduce the configuration of the database, so the costs can fit the OpenStreetMap US budget.

Are there any updates on it? (Assuming this quote talks about the overpass server.)

**Comment by mmd-osm** [comment]
> Assuming this quote talks about the overpass server

I don’t think this has anything to do with overpass, since none of the following applies:

> We have also made some optimizations on the database, for example evaluating the indexes and changing some SQL queries to be more efficient.

**Comment by andrewharvey** [comment]
Since #772 in v1.2.0 I'm noticing changesets are very fast to load and visualise.
